explainer.md

```diff
@@ -23,17 +23,33 @@ if the user agent is busy performing layout. The reason for this is that script
 runs in the same event loop as layout, and browser rendering semantics require
 previous layout and visual update to be done before running subsequent script.
 
-We propose a new concept, display locking, to assist developers with alleviating
-jank caused by DOM updates. Using display locking, the developer will be able to
-lock an element and its subtree, preventing visual updates. Then, the developer
-will be able construct the locked subtree’s DOM however they desire, and insert
-it into the DOM without any rendering cost or jank. After insertion, the element
-can be updated. These updates will be co-operative -- interleaved with other
-work such as running script or DOM updates outside of the locked subtree. The
+Creating more complex applications might also mean having a **bigger amount of things in a web page**,
+making the rendering costs to have all of the content to actually be part of the DOM
+to be prohibitively expensive.
+This drives some amount of web authors to *virtualization* instead,
+where they actually **refrain from putting things in the DOM**,
+keeping content in memory in some non-DOM data structure,
+with only the visible portion converted into DOM nodes and inserted into the document,
+recycling them in and out as needed.
+Popular examples include https://m.twitter.com/ and [react-window](https://github.com/bvaughn/react-window).
+However, this will **make the web app suffer** because a lot of things like find-in-page,
+accessibility, indexability, focus navigation, anchor links, etc.
+**depends on having things in the DOM**.
+
+We propose a new concept, **display locking**, to assist developers with **alleviating
+jank caused by DOM updates** and having some control on **when to pay rendering costs**,
+while also **allowing things that depend on the updates to get the up-to-date values** when urgent.
+
+Using display locking, the developer will be able to lock an element and its subtree,
+preventing visual updates.
+Then, the developer will be able construct the locked subtree’s DOM however they desire,
+and insert it into the DOM without any rendering cost or jank. After insertion, the element
+can be updated. These updates will be *co-operative* -- **interleaved with other
```

Comment:
"updated" here needs some explanation. Maybe "marked for update".

Reply:
Changed it a bit, and also the commit part. PTAL